baccmod/model.py

```diff
@@ -13,7 +13,7 @@
 """
 import numpy as np
 
-__all__ = ['BilinearGradient']
+__all__ = ['BilinearGradient', 'Gaussian2D']
 
 from astropy.modeling import Fittable2DModel, Parameter
 from astropy.modeling.functional_models import Gaussian2D
```

Comment:
If the goal is to use astropy models, importing some here only to allow import from baccmod is not natural. Remove this import and the entry in __all__.

We may want to provide an easier interface but I don't believe this is the way to do it.